src/components/bcourses/shared/MaintenanceNotice.vue

```diff
@@ -28,6 +28,10 @@
   </v-alert>
 </template>
 
+<script setup>
+import {mdiAlert} from '@mdi/js'
+</script>
+
 <script>
```

Comment:
If/when we start using the syntax of Vue 3 Composition API then this split-brain import setup goes away. https://vueschool.io/articles/vuejs-tutorials/options-api-vs-composition-api/